Packages/com.unity.inputsystem/InputSystem/Controls/InputControlPath.cs

```diff
@@ -558,15 +558,15 @@ private static bool StringMatches(Substring str, InternedString matchTo)
                         return true; // Wildcard at end of string so rest is matched.
 
                     ++posInStr;
-                    nextChar = char.ToLower(str[posInStr]);
```

Comment:
Could a fix be to convert to pass CultureInfo.InvariantCulture

E.g. 
nextChar =  char.ToLower(str[posInStr], CultureInfo.InvariantCulture);
or (if version allowed) :
nextChar =  char.ToLowerInvariant(str[posInStr]);

and a few lines down

Reply:
`ToUpperInvariant` is recommended here: https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings
I believe going in the other direction (lower) creates issues.

Reply:
@lyndon-unity @jamesmcgill I just updated whole branch to only use culture invariant char comparisons, this fixes the bug without changing to ToUpperInvariant(). I would suggest closing the ticket (as the bug is fixed) at this state and creating a new refactoring ticket for clean up the string ToLowerInvariant -> ToUpperInvariant situation. Maybe it even makes sense to do it in a bigger scale where we look at all our string storing and comparison situation

Reply:
I agree this is a good step and fixes the current but. We have captured the future step in a task here:
https://jira.unity3d.com/browse/ISX-1380